packages/web-components/src/components/va-accordion/va-accordion-item.tsx

```diff
@@ -84,21 +84,32 @@ export class VaAccordionItem {
   }
 
   render() {
+    // IE11 is unable to run onSlotChange
+    // Check needed to populate Accordion Item Header information
+    // When using slot="headline" to set the information
+    const ieSlotCheckTag =
+      this.el.querySelector('[slot="headline"]') !== null
+        ? this.el.querySelector('[slot="headline"]').tagName.toLowerCase()
+        : false;
+    const ieSlotCheckHeader =
+      this.el.querySelector('[slot="headline"]') !== null
+        ? this.el.querySelector('[slot="headline"]').innerHTML
+        : false;
+
```

Comment:
Ok, I think I'm understanding this now based on the previous comment. We've got these because `this.slotTag` and `this.slotHeader` aren't working in IE since the slotchange handler isn't running, right? And it also looks like IE is able to query by `[slot="headline"]`.

I know optional chaining doesn't work on IE, but could you try it here to see if our build handles it appropriately since its configured for `es5`? Something like:

```js
 const ieSlotHeader = this.el.querySelector('[slot="headline"]')?.innerHTML;
```

Reply:
@bkjohnson optional chaining works. I updated the PR and screenshots are here:
![Screenshot (24)](https://user-images.githubusercontent.com/11822533/152173887-7aa3b911-73b9-4de2-9b3e-c06afa9995aa.png)
![Screenshot (26)](https://user-images.githubusercontent.com/11822533/152173892-ff5066f9-3457-4486-b16e-655a533f45bc.png)
![Screenshot (25)](https://user-images.githubusercontent.com/11822533/152173894-712f1b06-0e53-4759-8a07-3aad4f86f05f.png)